core/chains/evm/logpoller/log_poller.go

```diff
@@ -794,7 +793,28 @@ func (lp *logPoller) blocksFromLogs(ctx context.Context, logs []types.Log, endBl
 	return lp.GetBlocksRange(ctx, numbers)
 }
 
-const jsonRpcLimitExceeded = -32005 // See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md
+// Defined as "failure to create transaction" in some references, but implemented as "results too large" in geth
+// See: https://github.com/ethereum/go-ethereum/blob/master/rpc/errors.go#L63
+const jsonRpcResultsTooLarge = -32003
+
+// This one is not implemented in geth by default, but is defined in EIP 1474 and is implemented by infura and many other
+// 3rd party rpc servers who do rate limiting. See: https://community.infura.io/t/getlogs-error-query-returned-more-than-1000-results/358/5
+const jsonRpcLimitExceeded = -32005 // See also: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md
```

Comment:
Technically with Go naming these should be `RPC`:
```suggestion
const jsonRPCResultsTooLarge = -32003

// This one is not implemented in geth by default, but is defined in EIP 1474 and is implemented by infura and many other
// 3rd party rpc servers who do rate limiting. See: https://community.infura.io/t/getlogs-error-query-returned-more-than-1000-results/358/5
const jsonRPCLimitExceeded = -32005 // See also: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md
```